### ISSUE TITLE
Vim tutorial

### ISSUE THREAD

**Issue by SheatNoisette** [Litarvan/epita-aled-content#1]
Maybe adding a short tutorial for Vim can be a good idea. It can be useful for users who don't like Emacs. Moreover, it's preinstalled on the school computers, unlike Nano.

**Comment by corrieriluca** [comment]
I agree, furthermore, Emacs is only used during two months of the first semester for OCaml.

**Comment by Litarvan** [comment]
Didn't know Emacs was only used the first two, thanks for the tips!
Vim is way better than Emacs when the use of plugin is prohibited (ofc Emacs+Evil is better on our personal computers), I'll try redacting a simple Vim tutorial, but the Tutor is already far better than anything I could do, I'll probably simply show the basics and then redirect to the Tutor for further use.

I'll still use Emacs with Spacemacs for the LaTeX tutorial, as it's probably one of the best LaTeX editor rn.
